examples/nidmm/continuous-acquire-graph-multiple-points.py

```diff
@@ -204,4 +204,6 @@ def on_close(event):
     error_message = rpc_error.details()
     if rpc_error.code() == grpc.StatusCode.UNAVAILABLE :
         error_message = f"Failed to connect to server on {server_address}"
+    elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
```

Comment:
We shouldn't be shipping unimplemented code to customers, so I don't think we should be promoting that error from our examples.

Reply:
If there is unimplemented function for some reason (like server handle not yet updated with new APIs introduced in a latest driver version; or we do update server handle & some APIs are deprecated) In such cases what would happen when the customer runs their existing code with those APIs which no longer have an implementation? will an exception be thrown? or will it exit without any message?

If it just exits then that is not a good experience as user wont know whether the program executed successfully or not. How about passing on the error status code as is (in else case) instead of explicitly calling out 'unimplemented functions'

Since examples are a point of 'getting started' for new users, they would retain/follow the skeleton of the example while writing their applications. Hence it would be good to somehow handle this.

Reply:
I think more simpler case which requires this handling would be what if client is using latest proto files and calls into older version of server binary? Calling functions that are not present in server UNIMPLEMENTED error too, and we should report this to the user. In case of such error, `rpc_error.details()` gives out empty string and we didn't used to print anything on console.

Reply:
@jonlugoatx , there is one more alternative to it. What if instead of printing "Function not implemented", we display the status code of the exception?
for eg.
![image](https://user-images.githubusercontent.com/78592845/116700381-58a86280-a9e4-11eb-86ce-cc7602abe173.png)

the output would look something like this.
![image](https://user-images.githubusercontent.com/78592845/116700458-6bbb3280-a9e4-11eb-8e57-885d54f5b7a5.png)

Reply:
@harsha-bhushan, if the user calls a function that is not implemented in the driver, we will return `NOT_FOUND`. But @doshirohan's example of the newer client calling an older server is valid. Looks like we need to check for both `NOT_FOUND` and `UNIMPLEMENTED`.

Reply:
I added check for `NOT_FOUND` as well

Reply:
NOTE: If this is approved, then same changes will be made in DCPower example.